src/commands/exile_commands.py

```diff
@@ -54,3 +57,53 @@ async def exile(
             await interaction.response.send_message(
                 error_message or f"Successfully exiled {user.mention}", ephemeral=True
             )
+
+    @bot.tree.command()
+    @discord.app_commands.checks.cooldown(
+        1, 86400, key=lambda i: (i.guild_id, i.user.id)
+    )
+    async def roulette(interaction: discord.Interaction):
+        """Test your luck, fail and be exiled..."""
+        exile_duration_options = [0, 1, 6, 12, 18, 24]
+        rand_choice = choice(exile_duration_options)
+
+        async with create_logging_embed(interaction) as logging_embed:
+            if rand_choice != 0:
+                reason = "roulette"
+                exile_duration = calculate_time_delta(f"{rand_choice}h")
+                error_message = await exile_user(
+                    logging_embed, interaction.user, exile_duration, reason
+                )
+
+                if error_message:
+                    logger.error(f"An error occurred: {error_message}")
+                    await interaction.response.send_message(
+                        "An error occurred while processing the command.",
+                        ephemeral=True,
+                    )
+                    return
+                else:
+                    await interaction.response.send_message(
+                        f"<@{interaction.user.id}> has tested their luck and has utterly failed! <@{interaction.user.id}> has been sent into exile for {rand_choice} hour(s).",
+                        ephemeral=False,
+                    )
+            else:
+                await interaction.response.send_message(
+                    f"<@{interaction.user.id}> has tested their luck and lives another day...",
+                    ephemeral=False,
+                )
+
+    @bot.tree.error
+    async def on_app_command_error(interaction: discord.Interaction, error):
```

Comment:
Nit: this probably should be moved to its own file instead of being combined with the exile commands, but we can move it later

Reply:
handled in https://github.com/naurffxiv/moddingway/pull/123